interfaces/evse_manager.yaml

```diff
@@ -49,8 +49,8 @@ cmds:
     arguments:
       reservation_id:
         description: >-
-          The reservation id (should be added to the TransactionStarted
-          event)
+          The reservation id (should be added to the TransactionStarted event). Negative value if there was
+          no specific reservation id for this evse (if there are 'global' reservations for 'any' evse).
```

Comment:
```suggestion
          The reservation id (should be added to the TransactionStarted event). Set tis to a negative value if there is 
          no specific reservation id for this evse but the evse should still move to a Reserved state because of total 
          global reservations.
```